knx/dpt/types.go

```diff
@@ -316,3 +316,211 @@ func (d DPT_12001) Unit() string {
 func (d DPT_12001) String() string {
 	return fmt.Sprintf("%d", uint32(d))
 }
+
+// DPT_13001 represents DPT 13.001 / counter value.
+type DPT_13001 int32
+
+func (d DPT_13001) Pack() []byte {
+	return packV32(int32(d))
+}
+
+func (d *DPT_13001) Unpack(data []byte) error {
+	var value int32
+
+	if err := unpackV32(data, &value); err != nil {
+		return err
+	}
+	*d = DPT_13001(value)
+
+	return nil
+}
```

Comment:
`return unpackV32(data, (*int32)(d))` would be much clearer.

Reply:
Done